build.gradle

```diff
@@ -1184,6 +1184,20 @@ task checkBasicStyle(group: 'Format') {
 
 assemble.mustRunAfter(clean)
 
+/**
+ * Execute the `assemble` task without generating any Javadoc or sources.jar files.
+ */
+task fastAssemble(group: 'Build') {
```

Comment:
I don't think this is working as intended.  It's also not building `framework-test/build/libs/framework-test-3.29.1-SNAPSHOT.jar`.  I think disabling the Javadoc tasks is disabling any task that depends on a Javadoc, too.  So in the end only `copyJarsToDist` is running, which doesn't depend on any Javadoc tasks.